features/multimedia.feature

```diff
@@ -1,27 +1,35 @@
 Feature: Multimedia
   In order to mange Multimedia
-  I need abe able to create and retrieve through the API.
+  I need abe able to create and retrieve through the API.'
+
   Scenario: Upload new multimedia file
     Given current authentication token
     And I attach "features/multimedia-test-image.png" to the request as upload
     When I request "/api/v1/multimedia/upload" using HTTP POST
     Then created response is received
-    And the response body contains JSON:
-    """
-      {"id": "01730e8d-fb8d-5afe-9be1-b621bacbb6dd"}
-    """
+    And remember response param "id" as "multimedia_id"
+
+  Scenario: Upload same multimedia file again
+    Given current authentication token
+    And I attach "features/multimedia-test-image.png" to the request as upload
+    When I request "/api/v1/multimedia/upload" using HTTP POST
+    Then created response is received
+
   Scenario: Download uploaded multimedia file
     Given current authentication token
-    And I request "api/v1/multimedia/01730e8d-fb8d-5afe-9be1-b621bacbb6dd" using HTTP GET
+    And I request "api/v1/multimedia/@multimedia_id@" using HTTP GET
     Then the response code is 200
-    And the "Content-Length" response header is 937
+    And the "Content-Length" response header is 1229
```

Comment:
Maybe we need use  https://github.com/Behatch/contexts
and `behatch:context:json`   and test json like in this example https://github.com/Behatch/contexts/blob/master/tests/features/json.feature